tasklist/config/envs/.env.database.elasticsearch

```
@@ -25,7 +25,7 @@ CAMUNDA_TASKLIST_DATABASE=${DATABASE}
 CAMUNDA_TASKLIST_ZEEBE_COMPATIBILITY_ENABLED=true
 
 # Operate Configuration
-CAMUNDA_OPERATE_TASKLIST_URL=http://${DATABASE_CONTAINER_NAME}:9200
+CAMUNDA_OPERATE_TASKLIST_URL=http://${TASKLIST_CONTAINER_NAME}:8080
```

```suggestion
CAMUNDA_OPERATE_TASKLIST_URL=http://localhost:8080
```
Since this is for WEB redirect we want the external URL here.